.github/workflows/flake8.yml

```diff
@@ -0,0 +1,24 @@
+name: run flake8 on source code
+
+on: push
```

Comment:
Let's move this to the same it CI job as yapf maybe? Otherwise starting one CI job per python linting tool will become a bit noisy in CI report.

Reply:
OK, will do